.ci/install_cloud_hypervisor.sh

```diff
@@ -17,12 +17,15 @@ arch=$("${cidir}"/kata-arch.sh -d)
 source "${cidir}/lib.sh"
 
 main() {
+	local buildscript="${katacontainers_repo_dir}/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh"
```

Comment:
why we are using the binaries? this will affect other CI jobs that installs clh

Reply:
This is exactly what we're already doing for cloud-hypervisor, @GabyCT, but instead of calling the script, we call `sudo -E PATH=$PATH make cloud-hypervisor-tarball`, which would then be translated to `${katacontainers_repo_dir}/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh" --build=cloud-hypervisor`.

This was introduced not so long ago (see: https://github.com/kata-containers/tests/pull/4463) and ensures that we rely on a single way to build cloud-hypervisor, respecting the needed flags, across all the different parts of the project (CIs, snap, kata-deploy, release binaries, etc).

Does this cover your concern?

Reply:
great thanks @fidencio for the explanation